static/app/views/performance/newTraceDetails/index.tsx

```diff
@@ -1,5 +1,7 @@
 import type React from 'react';
 import {
+  type Dispatch,
```

Comment:
nit: No need to import react types, you can just use the global `React.Dispatch`